38/documentation.html

```diff
@@ -30,7 +30,7 @@
     <!--#include virtual="toc.html" -->
   </div>
   <div class="right">
-    <!--//#include virtual="../includes/_docs_banner.htm" -->
```

Comment:
Interesting, shouldn't this have been done during 3.9.0 release?

Reply:
yeah, it should have been...